From 9033a8f8e884822e6dd3213800f539e0234534c3 Mon Sep 17 00:00:00 2001
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
Subject: [01/02] test_lic_files_chksum_presence_FAIL
From: John Doe <john.doe@somewhere.com>
X-Patchwork-Id: 123456
Message-Id: <b59fd4c5891e116865197f6a71186a99b893924b.1461723377.git.john.doe@somewhere.com>
Date: Thu, 7 Jul 2016 16:26:48 -0700

This patch will FAIL the test_lic_files_chksum_presence patchtest test.

---
diff --git a/LICENSE.bb b/LICENSE.bb
new file mode 100644
index 0000000..017f622
--- /dev/null
+++ b/LICENSE.bb
@@ -0,0 +1,6 @@
+LICENSE ?= "MIT"
+L IC_FILES_CHKSUM ?= "${@oe.utils.ifelse(d.getVar('LICENSE', True) == 'MIT', \
+                     'file://${COREBASE}/meta/COPYING.MIT;md5=3da9cfbcb788c80a0384361b4de20420', \
+                    '')}"
+
+

--
1.8.3.1

